pkg/server/entry.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2021 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/tektoncd/chains/proto"
+)
+
+func getEntry(uid string) (*proto.Entry, error) {
+	log.Printf("Getting entry with uid %s...", uid)
+	f := path(uid)
+	contents, err := ioutil.ReadFile(f)
```

Comment:
The `ioutil.ReadFile` makes it hard to unit test. Could this instead be https://pkg.go.dev/io#Reader?

Reply:
resolved as i've added the unit test :)